docs/api-docs/apps/guide/apps-05-oauth.md

```diff
@@ -87,7 +87,7 @@ GET /auth?code=qr6h3thvbvag2ffq&scope=store_v2_orders&context=stores/g5cd38 HTTP
 > ### Note
 >
 > * When your app receives a new token, any previously issued token is invalidated.
-> * As a best practice, your app should validate this list to ensure that it matches your app's needs, and fail if it does not. At this time, the user can not pick and choose scopes. The dialog presented to the user requires the user to approve all scopes or none.
+> * As a best practice, your app should validate the list of scopes to ensure that it matches your app's needs, and fail if it does not. At this time, the user can not pick and choose scopes. The dialog presented to the user requires the user to approve all scopes or none. For more information about available scopes, see [OAuth scopes](https://developer.bigcommerce.com/api-docs/getting-started/authentication/rest-api-authentication#oauth-scopes).
 > * The request comes from the client browser, rather than directly from BigCommerce. This allows you to use a non-publicly available auth callback URL while testing your app.

```

Comment:
> * The request comes from the client browser, rather than directly from BigCommerce. This request allows you to use a non-publicly available auth callback URL while testing your app.